crates/extensions/c8y_mapper_ext/src/operations/handlers/config_update.rs

```diff
@@ -47,14 +48,23 @@ impl OperationContext {
                 extra_messages: vec![],
             }),
             CommandStatus::Successful => {
+                let config_fragment = MqttMessage::new(
+                    sm_topic,
+                    set_c8y_config_fragment(
+                        &command.payload.config_type,
+                        &command.payload.remote_url,
```

Comment:
The URL can't be sent as-is to the cloud as this URL is the proxy URL, like `http://127.0.0.1:8001/c8y/inventory/binaries/4119377`. The url base must be changed to point to the tenant.

Reply:
Proxy conversion added in 97ad0bf9de2a4097bf8ae55dfad628602490710f